torch/csrc/generic/StorageMethods.cpp

```diff
@@ -179,6 +179,31 @@ static PyObject * THPStorage_(fromBuffer)(PyObject *_unused, PyObject *args, PyO
 }
 #endif
 
+static PyObject * THPStorage_(fromFile)(PyObject *_unused, PyObject *args, PyObject *keywds)
+{
+  HANDLE_TH_ERRORS
+  const char *filename;
+  Py_ssize_t size = 0;
+  int shared = 0, shared_mem = 0;
+  static char *kwlist[] = {"filename", "shared", "size", "shared_mem", NULL};
```

Comment:
I chose this naming to be the same as in Lua Torch. Anyway we should write description in docs. But if you think `shm` is better I'll change